diff --git a/configure b/configure
index 454c6c57fe..299db49c8d 100755
--- a/configure
+++ b/configure
@@ -7754,7 +7754,7 @@ case "$XPLATFORM" in
     *-g++*)
 	# Check gcc's version
 	case "$(${QMAKE_CONF_COMPILER} -dumpversion)" in
-	    8*|7*|6*|5*|4*|3.4*)
+	    9*|8*|7*|6*|5*|4*|3.4*)
 		;;
             3.3*)
                 canBuildWebKit="no"
@@ -8070,7 +8070,7 @@ g++*)
     3.*)
         COMPILER_VERSION="3.*"
         ;;
-    8*|7*|6*|5*|4.*)
+    9*|8*|7*|6*|5*|4.*)
         COMPILER_VERSION="4"
         ;;
     *)
